src/app/feed/feed-card/feed-card.component.html

```diff
@@ -31,7 +54,7 @@ <h3 class="lightbox-name">{{profileName}}</h3>
 					</span>
 				</a>
 			</div>
-			<div class="card-text">
+			<div class="card-text" (click)="showStyle = !showStyle;">
```

Comment:
This should not be here instead on complete box. Please try to stay as close to one issue as possible. :)